src/pages/Discover.tsx

```diff
@@ -34,7 +34,7 @@ const Discover: React.FunctionComponent<RouteComponentProps> = ({
         <Marketplace>
           <manifold-marketplace
             featured="piio"
-            link-format="/new/resource/:product"
+            product-link-format="/new/resource/?product=:product"
```

Comment:
If you look a few lines above this, we still have a click handler. This doesn't work currently with `react-router`